common/core/src/commonMain/kotlin/com/trafi/ui/theme/internal/Color.kt

```diff
@@ -11,4 +11,20 @@ import kotlin.jvm.JvmName
 
 expect inline class Color(val value: ULong)
 
+internal fun Color.alpha(value : Float): Color {
```

Comment:
Very clear Kotlin function, super nice job @mikasdo 👏 Good visibility modifier, using `require` to verify inputs, clear variable names 💪